schemas/profiles.v1.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "profiles",
+  "description": "profile data from vroom?",
```

Comment:
This is an unprocessed profile message emitted from Relay.

Reply:
oops, thanks